### PR TITLE
DS-882: Clean up TS reg api.

### PR DESCRIPTION
**Pull request by sherakama**
# READY FOR REVIEW

# Summary
- Just some clean up and revising while I was debugging
- This doesn't fix anything. Just changes things.

# Review By (Date)
- Whenever

# Criticality
-  Not

# Review Tasks

1. Visit each of the APIs and ensure they are returning properly formatted data
2. https://deploy-preview-969--stanford-alumni.netlify.app/api/travel-study/reg/internal-deposit
3. https://deploy-preview-969--stanford-alumni.netlify.app/api/travel-study/reg/post-extension
4. https://deploy-preview-969--stanford-alumni.netlify.app/api/travel-study/reg/pre-extension
5. https://deploy-preview-969--stanford-alumni.netlify.app/api/travel-study/reg/room-category
6. https://deploy-preview-969--stanford-alumni.netlify.app/api/travel-study/reg/trip-deposit
7. https://deploy-preview-969--stanford-alumni.netlify.app/api/travel-study/trips/
8. https://deploy-preview-969--stanford-alumni.netlify.app/api/travel-study/

### CODE REVIEW

File: src/api/travel-study/index.js [hunk 1]
```diff
@@ -6,49 +6,22 @@ const tripsCollection = async (req, res) => {
...
-    currentPage += 1;
-    requests.push(
-      storyblok.get(`cdn/stories/`, {
-        filter_query: {
-          component: {
-            in: 'trip',
-          },
+  let trips;
+  try {
+    trips = await storyblok.getAll(`cdn/stories`, {
```
Comment (sherakama) [inline src/api/travel-study/index.js:11]: Use get all instead of doing the paging ourselves.

File: src/api/travel-study/reg/internal-deposit.js [hunk 2]
```diff
@@ -16,7 +16,7 @@ const storyblok = new StoryblokClient({
  */
 export default async function handler(req, res) {
   res.setHeader('Access-Control-Allow-Origin', '*');
-  // res.setHeader('Content-Type', 'text/csv');
+  res.setHeader('Content-Type', 'text/csv');
```
Comment (sherakama) [inline src/api/travel-study/reg/internal-deposit.js:19]: Doesn't change how GG handles this but it should have been set after testing. It was commented out so others could see it in the browser for easy testing.

File: src/api/travel-study/reg/internal-deposit.js [hunk 3]
```diff
@@ -66,7 +66,7 @@ export default async function handler(req, res) {
         trip.content?.depositCost.replace(/\D/g, '');
       data.push([
         'prompt',
-        trip.content.tripId,
+        trip.content.tripId.trim(),
```
Comment (sherakama) [inline src/api/travel-study/reg/internal-deposit.js:69]: I noticed some bad copy/pasting in the fields and this clears that up. Not that it broke anything but it breaks my balance with the universe to see extra spaces.

**Comment by ericbakenhus** [comment]
https://deploy-preview-969--stanford-alumni.netlify.app/api/travel-study/trips/ seems to be returning a 500.

**Comment by sherakama** [comment]
> @sherakama [deploy-preview-969--stanford-alumni.netlify.app/api/travel-study/trips](https://deploy-preview-969--stanford-alumni.netlify.app/api/travel-study/trips/) seems to be returning a 500.

[DOH](https://github.com/SU-SWS/saa_alumni/pull/969/commits/a74667fcaa7105441d6139aa44ba29d911ae0ecd)

**Review by ericbakenhus — APPROVED** [review]
LGTM! 👍

**Comment by sherakama** [comment]
Thanks.

**Pull request merged**